providers/flagd/pkg/service/http/resolve-number_test.go

```diff
@@ -0,0 +1,154 @@
+package http_service
```

Comment:
These ought to be in a test package: `package http_service_test`

Reply:
updated